Modules/Module_Processing/Module_Population/xASL_stat_GetROIstatistics.m

```diff
@@ -31,6 +31,18 @@
 %                       0 = [0 0 0 0]
 %                       Can be useful for e.g. loading lesion masks outside the GM
 %                       (OPTIONAL, DEFAULT=1)
+%   x.S.bWMH           - boolean specifying if we should mask a ROI with pvWMH (by subtracting pvWMH in ASL resolution)
+%                        true = include WMH (no masking), false = exclude WMH (OPTIONAL, DEFAULT=false)
+%   x.S.bTissueMasking - value specifying which subject-wise tissue type we will mask with. Cannot be a vector
+%                        (loop outside this function for multiple tissue types)
+%                        'GM' = gray matter (GM)
+%                        'WM' = white matter (WM)
+%                        'WB' = whole brain (WB = GM+WM)
+%                        (REQUIRED, single value)
+%                        Cannot be a vector. Iterate outside this function over multiple x.S.Atlases-x.S.bTissueMasking combinations.

```

Comment:
A general question
can we give Atlases as {'TotalGM' 'Supratentorial' 'Supratentorial'} - with two names repeated as 
TissueMasking will be {'GM' 'WM' 'GM'}.

I understand that this is possible and it is good. Just asking if the code will work with that.

Reply:
Yes, xASL_stat_GetROIstatistics loops over x.S.Atlases and will take the same index from x.S.TissueMasking. Previously, we check that those have equal length. And if they don't make sense (e.g. TotalGM combined with 'WM') then it will return n/a to the tsv because it is empty.

Thinking about this, I need to check the variable name in the tsv. It should be TotalGM_GM Supratentorial_GM Supratentorial_WM in this example.

Reply:
Can you check and confirm before merging?